Index: opensprints/sensor.rb
===================================================================
--- opensprints/sensor.rb	(revision 42)
+++ opensprints/sensor.rb	(working copy)
@@ -1,7 +1,7 @@
 require 'socket'
 Kernel::require 'serialport.so' #windows is silly
 s = TCPSocket.new( "localhost", 5000 )
-sp = SerialPort.new('/dev/ttyUSB0', 115200, 8, 1, SerialPort::NONE)
+sp = SerialPort.new(0, 115200)
 t_start = Time.now.to_f
 old_cts_time = t_start
 old_dsr_time = t_start
